image-layout.md

```diff
@@ -163,16 +163,9 @@ Those tags will often be represented in an image-layout repository with matching
 ```json,title=Image%20Index&mediatype=application/vnd.oci.image.index.v1%2Bjson
 {
   "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.index.v1+json",
   "manifests": [
     {
-      "mediaType": "application/vnd.oci.image.index.v1+json",
-      "size": 7143,
-      "digest": "sha256:0228f90e926ba6b96e4f39cf294b2586d38fbb5a1e385c05cd1ee40ea54fe7fd",
-      "annotations": {
-        "org.opencontainers.image.ref.name": "stable-release"
-      }
-    },
```

Comment:
Thinking about this again, I would like to see `application/vnd.oci.image.index.v1+json` added to the media types which MUST be supported here.  Any consumer which gets to this point already understands the type (because they're reading one), so (for those implementations) an `application/vnd.oci.image.index.v1+json` reference will not meet the “unknown media-type” ignore condition.  The MUST entry will make it very clear (although I think it is already true) that compliant image-spec consumers will either:

1. Support `application/vnd.oci.image.index.v1+json` including `application/vnd.oci.image.index.v1+json` entries in `manifests`.
2. Not support `application/vnd.oci.image.index.v1+json` at all.

I don't see how “supports `application/vnd.oci.image.index.v1+json` but *not* `application/vnd.oci.image.index.v1+json` entries in `manifests`” is possible.